cli/azd/cmd/i18n/en.yaml

```diff
@@ -0,0 +1,92 @@
+availableCommands: "Available Commands"
+awesomeAzdUrl: "https://azure.github.io/awesome-azd"
+azdHats: "https://aka.ms/azure-dev/hats"
+azdShortHelp: "is an open-source tool that accelerates the process of getting your application onto Azure and helps you manage it once it's there."
+azdUpNodeJsMongo: "azd up --template todo-nodejs-mongo"
+azdUpTemplate: "azd up --template"
+cmdCommonFooter: "Use {{.AzdRun}} to view examples and more information about a specific command."
+cmdConfigShort: "Manage azd configurations (ex: default Azure subscription, location)."
+cmdDeployShort: "Deploy the application's code to Azure."
+cmdDownShort: "Delete Azure resources for an application."
+cmdEnvShort: "Manage your environment settings."
+cmdGroupTitleAbout: "About, help and upgrade"
+cmdGroupTitleConfig: "Configure and develop your app"
+cmdGroupTitleManage: "Manage Azure resources and app deployments"
+cmdGroupTitleMonitor: "Monitor, test and release your app"
```

Comment:
Does the package support using a more structured yaml document vs just a flat list of key/values?  It would be great if we could group of localization by command with meaningful properties.